sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/FilterPushdownBenchmark.scala

```diff
@@ -131,6 +134,15 @@ object FilterPushdownBenchmark extends BenchmarkBase with SQLHelper {
       }
     }
 
+    Seq(false, true).foreach { pushDownEnabled =>
+      val name = s"InMemoryTable Vectorized ${if (pushDownEnabled) s"(Pushdown)" else ""}"
+      benchmark.addCase(name) { _ =>
+        withSQLConf(SQLConf.IN_MEMORY_PARTITION_PRUNING.key -> s"$pushDownEnabled") {
```

Comment:
@wangyum . `FilterPushdownBenchmark` is not related to `Partition Pruning`, isn't it?
This benchmark case will be misleading. I'd like to have another benchmark for this `IN_MEMORY_PARTITION_PRUNING `.

Reply:
I think the InMemoryTable's partition same to Parquet RowGroup(@kiszk please correct if I'm wrong). We put them together and it's easy to compare performance.